Makefile

```diff
@@ -270,3 +270,21 @@ mixins:
 	$(CONTAINER_ENGINE) pull registry.svc.ci.openshift.org/ci/dashboards-validation:latest
 	$(CONTAINER_ENGINE) run --user=$(UID) --rm -v "$(CURDIR):/release:z" registry.svc.ci.openshift.org/ci/dashboards-validation:latest make -C /release/clusters/app.ci/prow-monitoring/mixins install all
 .PHONY: mixins
+
+# Runs e2e secrets generation and sync to clusters.
+#
+# Example:
+#   make secrets
+# First execute the following
+# echo -n "bw_password" > /tmp/bw_password
+# make kerberos_id=<your_kerberos_id> secrets
+secrets:
+	make kerberos_id=$(kerberos_id) ci-secret-generator
+	make job JOB=periodic-ci-secret-bootstrap
```

Comment:
We should use the `manual-trigger` tool in `test-infra` (or something similar to `cvp-trigger` in `ci-tools`) so we monitor these as they run.